mls-rs/src/client.rs

```diff
@@ -433,13 +435,15 @@ where
     ///
     /// A key package message may only be used once.
     #[cfg_attr(not(mls_build_async), maybe_async::must_be_sync)]
+    #[cfg_attr(all(feature = "ffi", not(test)), safer_ffi_gen::safer_ffi_gen_ignore)]
     pub async fn generate_key_package_message(
         &self,
         key_package_extensions: ExtensionList,
         leaf_node_extensions: ExtensionList,
+        maybe_now_time: Option<MlsTime>,
```

Comment:
Looking at the code, I find the interaction between an `Option<MlsTime>` and the actual key package generation confusing: the code really really wants an actual timestamp to use as the starting point for the lifetime.

So this braking API change would a chance to give it that: require people to pass in a `MlsTime` here and then the calls to `MlsTime::now()` below can go away! The conditional compilation code for Wasm will go away too.